usaspending_api/accounts/models.py

```diff
@@ -170,25 +167,20 @@ def totals(self):
             outlays[fiscal_year] += ab.gross_outlay_amount_by_tas_cpe
             obligations[fiscal_year] += ab.obligations_incurred_total_by_tas_cpe
         results = {
-            'outgoing': {
-                'outlays': outlays,
-                'obligations': obligations,
-                'budget_authority': budget_authority,
-            },
-            'incoming': {}
```

Comment:
Any chance we can disable JSON doing this? I haven't gotten that far into this PR yet, but this seems like a lot of unreadability coming our way...

Reply:
This isn't JSON, it is a python dictionary. For this specific example it is less-readable. We can thank previous devs who decided to make it easy to read. Previously there weren't many requirements on how dictionaries were formatted, so there are plenty of other examples where black's format is more readable

Reply:
20 files in I feel like the developers are winning most of these. I don't think going through each case will be productive; I just want to ask if you're sure that we can't just keep both formats and trust people to make smart decisions with dictionaries. I know black's whole idea is to be very strict, but this looks like a common thing we'll encounter.

Reply:
I suppose it is a personal preference. Yes, humans can make better decision for other humans over a program. Humans also make lots of mistakes, forget things, and have disagreements. Black is an opinionated code formatter. We'll remove the variability of the project's current code style in favor of a single format. If everyone hates it, we can re-evaluate.


```
    query = {
        "query": {
            "bool": {
                "must": [{
                    "match": {
                        "recipient_location_zip5": "abcde"
                    }
                }]
            }
        },
        "_source": ["award_id"]
    }```
vs
```
    query = {"query": {"bool": {"must": [{"match": {"recipient_location_zip5": "abcde"}}]}}, "_source": ["award_id"]}```

note: reducing the line count from 120 to ~90 will force more dictionaries to look like the "pretty print" style.